gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=e6d864e3b5bc05cc62041842b306383fc1fefcec359e70cebb1d470a6094ca82
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
```

Comment:
I don't think this should be removed, otherwise IIRC the Gradle wrapper will complain about it (and for good reason, really). Gradle distributes checksums on their [web page](https://gradle.org/release-checksums/#v7.6). You may also have inadvertently changed from the complete distribution to the binary-only distribution. I find the complete distribution helps with debugging, though I don't really have a stake in this any more.